src/lib/opencollective.js

```diff
@@ -2,8 +2,9 @@ import request from 'graphql-request';
 
 import cache from './cache';
 
-const baseUrl = 'https://opencollective.com/api/graphql';
-const baseUrlV2 = 'https://opencollective.com/api/graphql/v2';
+const opencollectiveBaseUrl = process.env.OPENCOLLECTIVE_BASE_URL;
```

Comment:
The convention accross our projects is to name this variable `API_URL`, maybe we can name it `OPENCOLLECTIVE_API_URL` here. The `/api/` path should then be removed.